cocotb/_build_libs.py

```diff
@@ -166,6 +166,9 @@ def _get_python_lib():
 _extra_cc_compile_args = ["-std=gnu99"] + _cc_warns
 _extra_cxx_compile_args = ["-std=c++11"] + _ccx_warns
 
+# [gh-1422]: Issue on RHEL6
```

Comment:
Can you clarify that this is actually not really a RHEL6 bug, but needed for all compilers without C++11 support?

```suggestion
# Make PRI* format macros available in compilers without C++11 support, e.g. on RHEL6.
```

Reply:
> Can you clarify that this is actually not really a RHEL6 bug, but needed for all compilers without C++11 support?

Is that really the case though? Isn't it a problem only when you actually *have* a C++11 compiler but an old libc?

Reply:
You're right, just adjust the message accordingly.